package.json

```diff
@@ -158,30 +168,34 @@
         "vscode:prepublish": "npm run compile",
         "compile": "npm install && npm run compile-ts && npm run compile-css && npm run compile-js",
         "compile-ts": "tsc -p ./",
-        "compile-css": "npm run compile-css-1 && npm run compile-css-2 && npm run compile-css-3 && npm run compile-css-4 && npm run compile-css-5",
+        "compile-css": "npm run compile-css-1 && npm run compile-css-2 && npm run compile-css-3 && npm run compile-css-4 && npm run compile-css-5 && npm run compile-css-6",
         "compile-css-1": "node-sass src/panels/tracker.scss -o out/panels",
         "compile-css-2": "node-sass src/panels/invoke.scss -o out/panels",
         "compile-css-3": "node-sass src/panels/create.scss -o out/panels",
         "compile-css-4": "node-sass src/panels/newwallet.scss -o out/panels",
         "compile-css-5": "node-sass src/panels/transfer.scss -o out/panels",
-        "compile-js": "npm run compile-js-1 && npm run compile-js-2 && npm run compile-js-3 && npm run compile-js-4 && npm run compile-js-5",
+        "compile-css-6": "node-sass src/panels/claim.scss -o out/panels",
+        "compile-js": "npm run compile-js-1 && npm run compile-js-2 && npm run compile-js-3 && npm run compile-js-4 && npm run compile-js-5 && npm run compile-js-6",
```

Comment:
Or a simpler way to compile these JS files?

Reply:
Yes, I can write a short script to enumerate the files that need compiling. Tracking with https://github.com/neo-project/neo-visual-tracker/issues/25